spec/system/search_system_spec.rb

```diff
@@ -61,6 +61,23 @@
     end
   end
 
+  it 'finds matching encryptable' do
+    login_as_user(:bob)
+    visit('/')
+
+    encryptable1 = encryptables(:credentials2)
```

Comment:
maybe create some more encryptables in same folder to make sure those are not rendered?
you might use fabricator for this
you then can add expectations that those are not rendered at all.

Reply:
I added two more credentials to the same folder via fabricator.